game_data.py

```diff
@@ -12,7 +16,21 @@
     }
 }
 
-players = {}
+players: List[Player] = []
 
 # start the server
 mud = MudServer()
+
+
+class Game():
```

Comment:
Maybe `GameState`. Possibly move `mud = MudServer()` into a `start` method as well, or some other way of moving state and moving parts out of the main namespace. It'll make it easier for testing in the future, and it's generally cleaner

Reply:
I've made a stab at this, not sure if it was what you had in mind.